src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -29,8 +29,8 @@
 (defn create-subscribers [request]
   (let [subscribers (extract-subscribers request)]
     (try
-      (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)
-      (present-response/success subscribers)
+      (let [insertions (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)]
```

Comment:
Yea so...

Basically yesql doesn't support insertions via a batch interface very well. It's in progress. This was the best thing I could come up with.

otherwise I think we should utilize Yesql queries.

`db/create-subscriber! subscriber-info` kind of thing